irohad/main/impl/on_demand_ordering_init.cpp

```diff
@@ -159,21 +159,24 @@ namespace iroha {
          *
          *   0 1 2
          * 0 o x v
-         * 1 x v .
+         * 1 v v .
          * 2 v . .
          *
-         * v, round 0 - kCurrentRoundRejectConsumer
-         * v, round 1 - kNextRoundRejectConsumer
-         * v, round 2 - kNextRoundCommitConsumer
-         * o, round 0 - kIssuer
+         * v, round 0,2 - kRejectRejectConsumer
+         * v, round 1,1 - kRejectCommitConsumer
+         * v, round 1,0 - kCommitRejectConsumer
```

Comment:
```suggestion
         * v, round 1,0 - kRejectCommitConsumer
```
0,0 ->reject 0,1 ->commit 1,0